intake/cli/server/server.py

```diff
@@ -160,6 +173,31 @@ def initialize(self, catalog, cache, auth):
         self._cache = cache
         self.auth = auth
 
+    def get(self):
```

Comment:
There is an argument that this belongs under the info/ endpoint:
- this location: because it's about a specific source
- in info: because we are only getting information

Reply:
Yeah, I actually put it in `/info` first and then moved it here. Feels like a tossup to me -- happy to follow your judgement here.